recipes/python-lzo/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+export CPPFLAGS="-I$PREFIX/include -I$PREFIX/include/lzo $CPPFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+
+$PYTHON setup.py install
```

Comment:
That is a minor thing anyway. You can test that in the next package iteration (updating or bumping the build number for some reason.)